app/controllers/OrganisationDetailsController.scala

```diff
@@ -29,12 +29,41 @@ import scala.concurrent.Future
 trait OrganisationDetailsController extends LisaBaseController {
   val rosmService:RosmService
 
+  def businessLables(businessStructure:  Option[BusinessStructure]): String = {
```

Comment:
I'd suggest not having BusinessStructure as an option here. Rather than throwing an exception in this method and catching it in the "get" method, check in the "get" method if it's a Some or None and only call this if it's a Some; otherwise do the redirect you're already doing anyway.